test/units/testsuite-54.sh

```diff
@@ -27,6 +27,26 @@ systemd-run -p LoadCredential=passwd:/etc/passwd \
             rm '${CREDENTIALS_DIRECTORY}/passwd' \
     && { echo 'unexpected success'; exit 1; }
 
+echo -n $RANDOM > /tmp/test-54-plaintext
```

Comment:
I guess there should be a check for the feature here? Maybe exit early if `+OPENSSL` is not in `systemctl --version` ?

Reply:
Please don't put a space after '>'.